.github/workflows/ci-cd.yml

```diff
@@ -176,10 +176,10 @@ jobs:
 
       - name: Deploy the latest documentation
         if: github.ref == 'refs/heads/main'
-        run: yarn deploy
+        run: yarn deploy --activate --verbose
 
       - name: Deploy the tagged documentation
         if: github.ref.type == 'tag'
-        run: yarn deploy
+        run: yarn deploy --activate --verbose
```

Comment:
Restored the flags to match the "before" code

https://github.com/ember-intl/ember-intl/blob/f9ca2e03a7d976cc5159dbe7cb5ac2a238f9cff7/.github/workflows/docs-tag.yml#L39-L42